framerail/src/app.d.ts

```diff
@@ -1,9 +1,103 @@
 // See https://kit.svelte.dev/docs/types#app
 // for information about these interfaces
 // and what to do when importing types
+
 declare namespace App {
   // interface Locals {}
-  // interface PageData {}
-  // interface Error {}
+  interface PageData {
+    /** Data about the site. */
+    site: {
+      site_id: string
+      [anySite: any]: unknown
+    }
+    /** Data about the page itself. */
+    page: {
+      page_id: string
+      page_created_at: string
+      page_updated_at?: string
```

Comment:
From what I understand, `field?` means the field itself may not be included. I think that the correct type is `string | null` since it's nullable, not optional, in the response you get from the API service.

Reply:
Not sure if i understand what you mean with `field?` .